vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -191,6 +191,12 @@ import hudson.AbortException
      * to the step call takes precedence.
      */
     'resources',
+    /**
+     * The path to which a volume should e mounted to. This will create an emptyDir volume
+     * with name 'volume'. This volume will be available at the same mount path in each container
```

Comment:
This is bit ambiguous. Name of the volume can be anything. But, the above description tells that it creates a volume with name "volume". Could you please elaborate on this?

Reply:
Good point, will do.

Reply:
Done